.env.template

```diff
@@ -7,3 +7,6 @@
 # SPDX-FileCopyrightText: 2023 Ahmed Sheta <ahmed.sheta@fau.de>
 
 GOOGLE_PLACES_API_KEY=
+
+FACEBOOK_APP_ID =
+FACEBOOK_APP_SECRET =
```

Comment:
Do these need to be set without a space?

Reply:
I thought optically grouping each api information alltogether would be better visually, but a space is not a problem, I'll delete it.

Reply:
Sorry there’s a misunderstanding. I agree that that makes it more readable. I meant the space before the = sign. The way the run_app.sh works right now will not correctly process a .env file with spaces before or after the = as it just appends the whole line. So either we need to remove the space here or adjust the script. In the long run adjusting the script to be more robust is probably better but feel free to do either.